shell/dialog/AddCustomBadgeDialog.vue

```diff
@@ -213,10 +213,10 @@ export default {
   <Card
     class="prompt-badge"
     :show-highlight-border="false"
-    :trigger-focus-trap="true"
   >
     <template #title>
       <h4 class="text-default-text">
+        ALEX
```

Comment:
Unfortunately, we'll need to remove the ALEX header 🙂

Reply:
Wanted to leave my mark on the codebase 🤣 🤣 🤣